superset/assets/javascripts/explorev2/stores/store.js

```diff
@@ -407,6 +407,21 @@ export const visTypes = {
     ],
   },
 
+  bullet: {
+    label: 'Bullet Chart',
```

Comment:
> the `bullet` object needs to contain `requiresTime: false`, so the time controls are not rendered.

Done. I still get the time controls, but perhaps it's due to testing with 0.14.1 instead of `master`. (Sorry.)